LondonBritishLibrary/orient/BLorient796.xml

```diff
@@ -32,9 +33,405 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <altIdentifier>
                      <idno>Wright 360</idno>
                   </altIdentifier>
-                  
+
                </msIdentifier>
-            </msDesc>
+               <msContents>
+                 <summary></summary>
+                 <msItem xml:id="ms_i1">
+                   <locus from="3r"></locus>
+                   <title type="complete" ref="LIT2317Senodo"></title>
+                   <textLang mainLang="gez"></textLang>
+
+                   <msItem xml:id="ms_i1.1">
+                     <locus from="3r"></locus>
+                     <title type="complete" ref="LIT2667Statut71Title" xml:lang="en">
+                     Orders of the Apostles, 71 titloi</title>
+                     <textLang mainLang="gez"></textLang>
+                   </msItem>
+                   <msItem xml:id="ms_i1.2">
+                     <locus from="4r"></locus>
+                     <title type="complete" xml:lang="en">Orders of the Apostles,
+                     here reckoned 81 in number.</title>
```

Comment:
This is very wrighty English :) I would adapt it, but if your policy is to change as little as possibly from the catalogue, that's fine.